diff --git a/exwm-ns.el b/exwm-ns.el
index 0118b80..920bf1b 100644
--- a/exwm-ns.el
+++ b/exwm-ns.el
@@ -34,8 +34,9 @@
    (cl-defgeneric seq-count (pred sequence)
      "Return the number of elements in SEQUENCE for which PRED returns non-nil."
      (let ((count 0))
        (seq-doseq (elt sequence)
          (when (funcall pred elt)
            (setq count (+ 1 count))))
        count))
-  (cl-defgeneric seq-contains (sequence elt &optional testfn)
+  (cl-defgeneric seq-contains-q (sequence elt &optional testfn)
      "Return the first element in SEQUENCE that is \"equal\" to ELT.
    \"Equality\" is defined by the function TESTFN, which defaults to `equal'."
      (declare (obsolete seq-contains-p "27.1"))
      (seq-some (lambda (e)
                  (when (funcall (or testfn #'equal) elt e)
                    e))
                sequence))